index.html

```diff
@@ -20,19 +20,20 @@
             <a href="/" class="logo">My Logo</a>
             <!-- <nav class="menu" id="icon"><i class="fa fa-bars "></i> -->
                 <nav class="navbar">
-                <ul class="navigations">
+                <ul class="navigations" id="navigations">
             
                     <li class="nav-item"> <a href="#"  class="nav-link"> Protfolio</a> </li>
                     <li class="nav-item"> <a href="#About"  class="nav-link">About</a> </li>
                     <li class="nav-item"> <a href="#contact"  class="nav-link" > Contacts</a> </li>
             </ul>
+
+            <!--this is hamburger-->
```

Comment:
- [ ] Great job on your hamburger menu its functioning perfectly and looks awesome however according to the requirements you should make sure that the hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins). To do that:
  - [ ] Please increase the font size of your mobile links
  - [ ] Please make sure the gap between each link is same as in figma design
  - [ ] Please change the use white icon for close(X) icon instead of black
  - [ ] [OPTIONAL] You can also use the property `filter:blur(3px)` and `opacity`on your mobile menu to give it the background effects similar to the template.

Figma design:

<img width="265" alt="Screenshot 2023-04-18 at 10 00 48 PM" src="https://user-images.githubusercontent.com/44798044/232852316-d690098a-3df2-4e88-a7ea-c96b05ea7362.png">

Your design:

<img width="367" alt="Screenshot 2023-04-18 at 10 01 10 PM" src="https://user-images.githubusercontent.com/44798044/232852347-f559c8d1-eab9-4eeb-8d8d-a2fb07cd7108.png">